animation_nodes/nodes/rotation/convert_rotations.py

```diff
@@ -66,30 +99,74 @@ def draw(self, layout):
 
     def drawLabel(self):
         for item in conversionTypeItems:
-            if self.conversionType == item[0]: return item[1]
+            if self.conversionType == item[0]:
+                if self.useList:
+                    return listTypeItems[item[0]]
```

Comment:
You can just leave this method as it was.

Reply:
then, how to change the label draw when chenging to list type? can you help me?